manifests/t/Teradata/TTUODBC/20.00.12.00/Teradata.TTUODBC.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Teradata.TTUOdbc
```

Comment:
Please update file name and path to match this line.

Reply:
Yes. We have fixed the issue but need to recreate the manifest files with the correct names. This will be done in the morning.